inc/notificationevent.class.php

```diff
@@ -160,7 +174,8 @@ static function raiseEvent($event, $item, $options = [], $label = '') {
                   $data,
                   $notificationtarget->setEvent($eventclass),
                   $template,
-                  $notify_me
+                  $notify_me,
+                  $emitter
```

Comment:
As far as I understand, `$emitter` will always be null here, so it is never used

Reply:
I see what you mean, it's the github diff which stop at a weird place and make it looks like `$emitter` is in the `else` block but that's not the case:
![image](https://user-images.githubusercontent.com/42734840/83641537-f82f5c00-a5ad-11ea-9f94-c4ecb68bd00e.png)

Reply:
Oh, OK, indeed, you're right; forgive me :)